zanata-war/src/main/java/org/zanata/action/CopyTransAction.java

```diff
@@ -52,7 +54,7 @@
  */
 @Name("copyTransAction")
 @Scope(ScopeType.CONVERSATION)
-public class CopyTransAction implements Serializable, ProgressBar {
```

Comment:
Ah, I figured it out. Just didn't occur to me that `CopyAction` extends `ProgressBar`.